platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxSymbolCountActivity.java

```diff
@@ -0,0 +1,180 @@
+import com.mapbox.services.commons.geojson.Feature;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.List;
+
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.iconImage;
+
+/**
+ * Demo's query rendered features on Symbols
+ */
+public class QueryRenderedFeaturesBoxSymbolCountActivity extends AppCompatActivity {
+    private static final String TAG = QueryRenderedFeaturesBoxSymbolCountActivity.class.getSimpleName();
+
+    public MapView mapView;
+    private MapboxMap mapboxMap;
+
+    private Toast toast;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_query_features_box);
+        setupActionBar();
+
+        final float density = getResources().getDisplayMetrics().density;
+
+        final View selectionBox = findViewById(R.id.selection_box);
+
+        //Initialize map as normal
+        mapView = (MapView) findViewById(R.id.mapView);
+        mapView.onCreate(savedInstanceState);
+        mapView.getMapAsync(new OnMapReadyCallback() {
+            @SuppressWarnings("ConstantConditions")
+            @Override
+            public void onMapReady(final MapboxMap mapboxMap) {
+                QueryRenderedFeaturesBoxSymbolCountActivity.this.mapboxMap = mapboxMap;
+
+                //Add a symbol layer (also works with annotations)
+                try {
+                    mapboxMap.addSource(new GeoJsonSource("symbols-source", readRawResource(R.raw.test_points_utrecht)));
+                } catch (IOException e) {
+                    Log.e(TAG, "Could not load geojson: " + e.getMessage());
+                    return;
+                }
+                mapboxMap.addImage("test-icon", BitmapFactory.decodeResource(getResources(), R.drawable.default_marker));
+                mapboxMap.addLayer(new SymbolLayer("symbols-layer", "symbols-source").withProperties(iconImage("test-icon")));
+
+
+                selectionBox.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        //Query
+                        int top = selectionBox.getTop() - mapView.getTop();
+                        int left = selectionBox.getLeft() - mapView.getLeft();
+                        RectF box = new RectF(left, top, left + selectionBox.getWidth(), top + selectionBox.getHeight());
+                        Log.i(TAG, String.format("Querying box %s", box));
+                        List<Feature> features = mapboxMap.queryRenderedFeatures(box, "symbols-layer");
+
+                        //Show count
+                        if (toast != null) {
+                            toast.cancel();
+                        }
+                        toast = Toast.makeText(
```

Comment:
For future situations, use a [Snackbar](https://developer.android.com/reference/android/support/design/widget/Snackbar.html) instead? Alternative link to a dutch [Snackbar](https://www.google.be/imgres?imgurl=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2F6%2F6b%2F2015_1016_Snackbar_Kwalitaria_Leidsenhage.jpg&imgrefurl=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FSnack_bar&docid=UH_AaNdXhprA2M&tbnid=vZZy_oloLwNoAM%3A&w=2560&h=1647&bih=672&biw=1074&ved=0ahUKEwiro_vU1NLPAhXBB8AKHY3OB8UQMwgzKAAwAA&iact=mrc&uact=8).